api/queries_repo.go

```diff
@@ -943,7 +957,8 @@ type RepoResolveInput struct {
 	Assignees  []string
 	Reviewers  []string
 	Labels     []string
-	Projects   []string
+	ProjectsV1 bool
```

Comment:
So, in my opinion, the previous approach of "accepting aggregate projects" and "returning v1 and v2 separately" signified something funny. The result was that we'd have to prop drill the support variable everywhere like so: https://github.com/cli/cli/compare/trunk...cli-9430#diff-5c07fcc22c80bad9356b4c3e1c794101855bcac70b031572c2084dc6d2023abaR872

I think it makes a lot more sense to make the decision as high up the stack as we can and then use types to correctly model expecations.

Not also that this changed to a `bool` because the values were never used.